ui/imports/utils/SubscriptionBroker.qml

```diff
@@ -0,0 +1,197 @@
+import QtQuick 2.15
```

Comment:
One more thought about that file - what about moving to StatusQ? It's very generic tool deserving to be there imo.

Reply:
Done. I've moved it to StatusQ. Together with the Subscription.qml